portal.go

```diff
@@ -1176,6 +1178,15 @@ func (portal *Portal) CreateMatrixRoom(user *User, groupInfo *types.GroupInfo, i
 	return nil
 }
 
+func (portal *Portal) addToSpace(spaceID id.RoomID, portalID id.RoomID) {
+
+	parentSpaceContent := make(map[string]interface{})
+	parentSpaceContent["via"] = []string{"matrix.wounn.xyz"}
```

Comment:
This needs to be the bridge's homeserver URL

Reply:
Yup. Should have fixed that when this commit was pushed.

From MSC 1772
> `via` key which gives a list of candidate servers that can be used to join the room

It means these servers will be used to bootstrap the join request ([permalink](https://matrix.to/#/!LPqOcoJQuGkweiTedq:riot.ovh/$H5vDqjNncsKd-2K4OpD2nbteji_ZPnWmtjeaLd1FJRg?via=matrix.org&via=envs.net&via=mozilla.org) to comment in spaces:riot.ovh)

The bridge's homeserver should be part of via - yes. Should be include the user's homeserver too if it is different from the bridge's homeserver? @HelderFSFerreira we'll try testing this on a federated instance.

Reply:
I've added the bridge home server as the value of the via.
Are we sure that we want to have the client's home server in there too?